akka-actor-typed-tests/src/test/scala/docs/akka/typed/InteractionPatternsSpec.scala

```diff
@@ -80,7 +80,7 @@ class InteractionPatternsSpec extends ScalaTestWithActorTestKit with AnyWordSpec
       val probe = createTestProbe[CookieFabric.Response]()
       // shhh, don't tell anyone
       import scala.language.reflectiveCalls
-      val context = new {
+      val context: { def self: ActorRef[CookieFabric.Response] } = new {
```

Comment:
Scala 3 doesn't infer refinements / strucutral types (it seems; I don't have a reference at hand)

Reply:
This is just some sneaky stuff to make a sample look like something it isn't, this is fine. :)

Reply:
> Scala 3 doesn't infer refinements / structural types (it seems; I don't have a reference at hand)

No reference either, but I know this to be true also.